src/helpers/predict_methods.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+from onnxruntime import InferenceSession
+from jaqpotpy.datasets.molecular_datasets import JaqpotpyDataset
+from src.helpers.recreate_preprocessor import recreate_preprocessor
+
+
+def predict_onnx(model, dataset: JaqpotpyDataset, request):
+    sess = InferenceSession(model.SerializeToString())
+    onnx_prediction = sess.run(
+        None, {"float_input": dataset.X.to_numpy().astype(np.float32)}
+    )
+    if len(request.model['dependentFeatures']) == 1:
+        onnx_prediction = onnx_prediction[0].reshape(-1, 1)
```

Comment:
While we're on it 😅  here we can use a for as well, some models have more than 1 predicted label, right?

Reply:
It's ok as it is for multi-output models

Reply:
To make it more evident, the reshape need to be applied only in single-output models, because the prediction array is one-dimensional (while we need it to be two-dimensional). Instead, the prediction of a multi-output model is by default 2-dimensional, so we do not need to reshape it

Reply:
Can we add a small comment explaining this?

Reply:
sure